Rover/sailboat.h

```diff
@@ -75,6 +75,14 @@ class Sailboat
         USE_MOTOR_ALWAYS = 2
     };
 
+    // states of SAIL_CONTROL_TYPE parameter and sail_control_type variable
+    enum SailControlType : uint8_t {
```

Comment:
```suggestion
    enum class SailControlType {
```

Reply:
using enum class instead of plain enum gives this error:

![image](https://user-images.githubusercontent.com/2212793/174702054-e4b4588d-853e-4f1b-bb0d-78c7f5a39bd0.png)

despite enum class being [preferred](https://stackoverflow.com/questions/18335861/why-is-enum-class-preferred-over-plain-enum), using plain enum seems to be the simpler solution in this case.

Reply:
I think you would have to cast the param:
```
switch (SailControlType(sail_control_type)) {
```
but as you say it does not make a huge difference either way.